src/overlays/actors/ovl_Oceff_Wipe5/z_oceff_wipe5.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_oceff_wipe5.c
  * Overlay: ovl_Oceff_Wipe5
- * Description:
+ * Description: Sonata/Lullaby/Bossa Nova/Elegy/Oath Ocarina Effect
```

Comment:
And technically Saria's Song, but we probably don't need to say that.

Reply:
Yeah, I left it out because Saria's Song uses the Sonata effect, so it's a little bit redundant. It's not like there's a dedicated Saria's Song effect like the other 5; it's just reusing an effect that already exists.

Reply:
Er... is it? I thought I saw a different `ofeff_wipe#` with Saria's name on it

Reply:
That's correct. One of the wipes is the *old* Saria's Song effect from Ocarina of Time; this wipe is completely unused. When you play Saria's Song in Majora's Mask, though, it shows a wipe5.